docs/apis.asciidoc

```diff
@@ -0,0 +1,50 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/java[elastic.co]
+endif::[]
+
+[[apis]]
+== Tracing APIs
+
+There are three different ways enhance the out-of-the-box instrumentation of the Java agent with manual instrumentation:
+
+. <<public-api>> +
+  A simple and stable API that is most native to the agent.
+  Contains annotations to declaratively create spans.
+. <<opentelementry-bridge>> +
+  A vendor neutral API.
+  If you plan to do a lot of manual instrumentation and want to reduce vendor lock-in this is probably what you're looking for.
+. <<opentracing-bridge>> +
+  A vendor neutral API that is discontinued in favor of OpenTelemetry.
+
+[float]
+[[apis-operation-modes]]
+== Operation Modes
+
+All APIs allow for different operation modes in combination with the Elastic APM agent
+
+Noop::
++
+--
+If the agent is not installed, the APIs are in noop mode and do not actually record and report spans.
+--
+
+Mix and Match::
++
+--
+If you want to leverage the auto instrumentation of Elastic APM,
+but also want to create custom spans or use the API to add custom labels to the spans created by Elastic APM,
+you can just do that.
+--
+
+Manual instrumentation::
++
+--
+If you don't want Elastic APM to auto-instrument known frameworks,
+but instead only rely on manual instrumentation,
+disable the auto instrumentation setting the configuration option <<config-instrument,`instrument`>> to `false`.
+--
+
+include::./api-elastic.asciidoc[]
+include::./api-opentelemetry.asciidoc[]
+include::./api-opentracing.asciidoc[]
```

Comment:
In the docs preview this doesn't seem to construct the side menu properly

Reply:
doc should be good now, required to #2519 + similar changes to `1.x` in order to allow docs build job to stop complaining about broken links.